### ISSUE TITLE
Investigate why Playwright/Axe-Core doesn't recognize <title> tags

### ISSUE THREAD

**Issue by JCantu248** [cisagov/XFD#574]
# 🐛 Summary #

Playwright/Axe-Core accessibility tests are throwing a violation (document-title) that the <title> element of the HTML page is empty. It is not empty, so we need to figure out why the error is being thrown. 

## To reproduce ##

Run Playwright tests in the CLI against Staging and observe the errors being thrown. 

## Expected behavior ##

There should be no error or violation thrown.
